bool Standstill
bool CpuOK
bool EncoderOK
bool SafeSpeed
bool LeftLidarActive
bool RightLidarActive
bool LeftEmergencyStopStatus
bool RightEmergencyStopStatus
bool LeftScannerContaminationError
bool RightScannerContaminationError
bool LeftScannerContaminationWarning
bool RightScannerContaminationWarning
bool RightScannerReducedSpeedZoneStatus
bool RightScannerSoftStopZoneStatus
bool RightScannerEmergencyZoneStatus
bool LeftScannerReducedSpeedZoneStatus
bool LeftScannerSoftStopZoneStatus
bool LeftScannerEmergencyZoneStatus
bool LeftScannerAppError
bool LeftScannerDeviceError
bool RightScannerAppError
bool RightScannerDeviceError
bool LeftScannerMonitoringCaseValid
bool RightScannerMonitoringCaseValid
